suites/portal/GWASUIAppTest.js

```diff
@@ -169,10 +185,23 @@ Scenario('Submit workflow Dichotomous Outcome - Dichotomous Covariate Phenotype
     I.seeElement(gwas.props.SubmitDialogBox);
     const jobName = gwas.do.enterJobName();
     await gwas.do.submitJob();
-    gwas.do.checkJobStatus();
+    gwas.do.goToJobStatus();
+    await gwas.do.checkStatus();
+});
+
+Scenario('GWAS Result App @GWASUI @manual', async ({ 
+    I, home, users, gwas 
+}) => {
+    I.useWebDriverTo('set window size', async ({ browser }) => {
+        browser.setWindowSize(1920, 1080);
+    });
+    home.do.goToHomepage();
+    await home.complete.login(users.mainAcct);
+    gwas.do.goToResultPage();
+    await gwas.do.checkStatus();
```

Comment:
Yes the test will fail because [here](https://github.com/uc-cdis/gen3-qa/pull/951/files#diff-e92a36e3422637e995cd59eb56c63772f8ff9379bb1e330ee0581dbe47a0ca4fR152) is looking at the first executed workflow. If it doesnt find the workflow, it will fail.